src/components/SecretsDeleteModal/SecretsDeleteModal.scss

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.deleteModal {
```

Comment:
let's revisit these styles later as part of the design updates and as these components are used

some general rules to keep in mind (applies to other scss files in this PR too):
- use default Carbon styles as much as possible
- when overriding styles we should aim to do it in a way that's applied consistently across the experience
- use design tokens instead of hardcoding colours etc.
- text should be left aligned
- overrides for specific cases should be very limited

Reply:
Most of the styles in this file are in regards with either adjusting the modal to fit the container or make the text in the modal bigger because the default font-size in this particular case is too small to read although I will make the appropriate changes & use design tokens